_posts/2016-07-27-custom-form-controls-in-angular-2.md

```diff
@@ -558,7 +558,7 @@ Next, we somehow have to pass these values to our `validateCounterRange(c: FormC
 {% highlight js %}
 {% raw %}
 export function createCounterRangeValidator(maxValue, minValue) {
-  return function validateCounterRange(c: FormControl) => {
+  return function validateCounterRange(c: FormControl) {
```

Comment:
I understand... my comment was suggesting that a closure is best represented (in ng2 codebases) with a fat-arrow syntax; as is seems (IMO) more clean.